_commands/basics/uniq.md

```diff
@@ -2,23 +2,39 @@
 ---
 
 uniq
---
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
+------
+
+`uniq`, often used with  `sort`, allows a user to report, or filter, repeated lines that are adjacent in a file. Some common and useful flags include -d or -u for listing lines that are duplicated or that are unique, respectfully. As well as -c for counting the number of times a line was dupicated.
 
 <!-- minimal example -->
 ~~~ bash
-*short* example of command usage and output
+$ sort in.txt | uniq -c 
+  2 line1
+  3 line2
+  1 line3
 ~~~
 
 <!--more-->
 
 ### Useful Options / Examples
+Often used with sort, uniq will count the number of times a unique line appears in a file when given the -c flag
 
 #### Example command
+~~~ bash
+$ sort in.txt | uniq -c 
+  2 line1
+  3 line2
+  1 line3
+~~~
 
 ##### Break it down
+You can also use uniq to list only lines that are duplicates (-d flag) or that are unique (-u flag) 
```

Comment:
"-d" needs backticks around it so it renders as code. ex: "-d" -> "`-d`"

Same thing for "`-u`"